### PR TITLE
Revise the version for Microsoft.AspNetCore.Server.IISIntegration.Tools

### PR DESCRIPTION
**Pull request by pranavkm**
Additionally fix project.json warnings

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @muratg \ @moozzyk

**Comment by Tratcher** [comment]
I assume this was intended for release, not dev?

**Comment by pranavkm** [comment]
Yup.

**Comment by muratg** [comment]
:shipit: in Release :)

File: makefile.shade [hunk 1]
```diff
@@ -2,6 +2,7 @@
 var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
+-BuildQuality = "";
```
Comment (moozzyk) [inline makefile.shade]: Why is this needed?

Comment (pranavkm) [inline makefile.shade]: This declares the BuildQuality for the entire repo. Since the two projects in this repo have different build quality, this resets it.

Comment (moozzyk) [inline makefile.shade]: I see. Thanks for the explanation. I see this being added to KoreBuild but did not really look to closely to understand what it does...
